tests/datasets/test_errors_processor.py

```diff
@@ -204,7 +216,7 @@ def test_error_processor() -> None:
     expected_result = {
         "org_id": 3,
         "project_id": 300688,
-        "timestamp": datetime(2020, 1, 30, 22, 48, 49, 894703),
+        "timestamp": error_timestamp.replace(tzinfo=None),
```

Comment:
Doesn't this already not have tzinfo?

Reply:
It did in a previous version. I did a cleanup after that to simplify this and I left this. behind
https://github.com/getsentry/snuba/pull/911. removes it